Microsoft.Toolkit.Win32.UI.XamlHost/UWPTypeFactory.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using xaml = Windows.UI.Xaml;
```

Comment:
This is inconsistent with other using shortcuts

Reply:
Agreed. I'll fix it.